Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerListModel.cpp

```diff
@@ -850,6 +851,34 @@ namespace AzToolsFramework
             return false;
         }
 
+        // Check if we're in focus mode
+        auto entityContextId = AzFramework::EntityContextId::CreateNull();
+        EditorEntityContextRequestBus::BroadcastResult(entityContextId, &EditorEntityContextRequests::GetEditorEntityContextId);
+        AZ::EntityId focusRoot = m_focusModeInterface->GetFocusRoot(entityContextId);
+        if (focusRoot.IsValid())
+        {
+            // Only allow reparenting the selected entities if they are all under the same instance.
+            // We check the parent entity separately because it may be a container entity and
+            // container entities consider their owning instance to be the parent instance
```

Comment:
nit: Since the same comments are included in the function header, these comments can be skipped.